test/e2e/e2e_test.go

```diff
@@ -20,69 +20,32 @@ package e2e
 
 import (
 	"flag"
-	"fmt"
-	"path"
 	"testing"
 	"time"
 
-	"github.com/onsi/ginkgo"
-	ginkgoconfig "github.com/onsi/ginkgo/config"
-	"github.com/onsi/ginkgo/reporters"
+	"github.com/onsi/ginkgo/v2"
 	"github.com/onsi/gomega"
 	"k8s.io/apimachinery/pkg/util/wait"
 
 	"github.com/cert-manager/cert-manager/pkg/logs"
-	"github.com/cert-manager/cert-manager/test/e2e/framework"
 	_ "github.com/cert-manager/cert-manager/test/e2e/suite"
 )
 
-var featureGates string
-
 func init() {
 	logs.InitLogs(flag.CommandLine)
-	framework.DefaultConfig.AddFlags(flag.CommandLine)
-
-	// Turn on verbose by default to get spec names
-	ginkgoconfig.DefaultReporterConfig.Verbose = true
-	// Turn on EmitSpecProgress to get spec progress (especially on interrupt)
-	ginkgoconfig.GinkgoConfig.EmitSpecProgress = true
-	// Randomize specs as well as suites
-	ginkgoconfig.GinkgoConfig.RandomizeAllSpecs = true
+	cfg.AddFlags(flag.CommandLine)
 
 	wait.ForeverTestTimeout = time.Second * 60
 }
 
 func TestE2E(t *testing.T) {
 	defer logs.FlushLogs()
-	flag.Parse()
```

Comment:
How come this has been removed?

Reply:
Normally, the testing package calls flag.Parse itself (https://groups.google.com/g/golang-nuts/c/P6EdEdgvDuc)